### ISSUE TITLE
predict the activity of compounds

### ISSUE THREAD

**Issue by jia-huang** [IBM/materials#39]
I want to use the fine-tuned model to predict the activity of compounds. Where can I find the code and commands for this part?

**Comment by jia-huang** [comment]
about smi-ted

**Comment by jia-huang** [comment]
> > Hi [@jia-huang](https://github.com/jia-huang) ! Thanks for your interest in using the FM4M. You can find all the code needed to fine-tune the SMI-TED model at the following path: https://github.com/IBM/materials/tree/main/models/smi_ted/finetune
> > The code supports regression, classification, and multi-output classification tasks. The necessary scripts are located in the folder. To get started, simply create a new folder for your specific task and copy the script from one of the existing datasets that best fits your needs.`smi_ted_light`
> > You'll also need to place your task data inside the finetune folder (or another location within the directory) and update the argument in your script, as shown here:`` materials/models/smi_ted/finetune``data_root ``
> > --data_root '../../moleculenet/freesolv'
> > You can also find the pre-trained model on HuggingFace: https://huggingface.co/ibm-research/materials.smi-ted
> > Please let me know if you have any questions or run into any issues.



> Hi [@jia-huang](https://github.com/jia-huang) ! Thanks for your interest in using the FM4M. You can find all the code needed to fine-tune the SMI-TED model at the following path: https://github.com/IBM/materials/tree/main/models/smi_ted/finetune
> 
> The code supports regression, classification, and multi-output classification tasks. The necessary scripts are located in the `smi_ted_light` folder. To get started, simply create a new folder for your specific task and copy the script from one of the existing datasets that best fits your needs.
> 
> You'll also need to place your task data inside the finetune folder (or another location within the `materials/models/smi_ted/finetune` directory) and update the `data_root` argument in your script, as shown here:
> 
> --data_root '../../moleculenet/freesolv'
> You can also find the pre-trained model on HuggingFace: https://huggingface.co/ibm-research/materials.smi-ted
> 
> Please let me know if you have any questions or run into any issues.

I have already fine-tuned the model. Now I have a compound library without known labels, and I want to use the fine-tuned model to make predictions on this library. I no longer need to fine-tune the model. What should I do?

**Comment by vichShir** [comment]
Hi @jia-huang ! You can use your fine-tuned model the same way you would a pre-trained one from the `inference` folder. We've made some [notebook examples](https://github.com/IBM/materials/tree/main/models/smi_ted/notebooks) available that you can also easily test your models. 

All you need to do is update the `ckpt_filename` option to use your fine-tuned checkpoint file.
```python
model_smi_ted = load_smi_ted(
    folder='../inference/smi_ted_light',
    ckpt_filename=FINETUNED_CHECKPOINT
)
```
